mbed_greentea/mbed_report_api.py

```diff
@@ -672,11 +734,19 @@ def exporter_html(test_result_ext, test_suite_properties=None):
                                                     platform,
                                                     toolchain,
                                                     test_results)
+                
+                # Loop through the test cases and count the passes and failures
+                for index, (testcase_result_name, testcase_result) in enumerate(test_results['testcase_result'].iteritems()):
+                    test_results['single_test_passes'] += testcase_result['passed']
+                    test_results['single_test_count'] += 1
 
                 result_class = get_result_colour_class(test_results['single_test_result'])             
                 this_row += result_cell_template % (result_class,
                                                     result_div_id,
                                                     test_results['single_test_result'],
+                                                    int((test_results['single_test_passes']*100.0)/test_results['single_test_count']),
+                                                    test_results['single_test_passes'],
```

Comment:
Hi
Please, have a look on https://github.com/ARMmbed/greentea/issues/236
This new division is crashing several non regression test sessions on our test bench....
Thx